Tools/cases_generator/generators_common.py

```diff
@@ -211,6 +212,37 @@ def py_stack_ref_from_py_object_new(
         # unused portions of the stack to NULL.
         stack.flush_single_var(self.out, target, uop.stack.outputs)
 
+    def stack_entry(
```

Comment:
I'm not sure that the stack calculations are correct here, and I don't think it will be able to handle code that writes to multiple results, like `UNPACK...`.

If we go back to using arrays, we can remove this.